src/tests.rs

```diff
@@ -0,0 +1,72 @@
+use failure::{err_msg, Error};
+use har;
+use serde_json::Value;
+
+use ::kubernetes_api::core::v1::NamespaceList;
+
+fn entries(ahar: &har::Har) -> Result<&Vec<har::v1_3::Entries>, Error> {
+    match &ahar.log {
+        har::Spec::V1_3(log) => Ok(&log.entries),
+        _ => Err(err_msg("unexpected har version")),
+    }
+}
+
+fn response(entry: &har::v1_3::Entries) -> Result<String, Error> {
+    // https://github.com/rust-lang/rust/issues/46871
+    // let body = &entry.response.content.text?;
+    let body = match &entry.response.content.text {
```

Comment:
Rust golf:
```rust
let body = entry.response.content.text.ok_or(err_msg("no response body"))?;
```

Reply:
I could have sworn I tried that and hit a conversion issue rooted in the result wrapper... but no, it works. Fixing.

Reply:
```error[E0507]: cannot move out of borrowed content
  --> src\tests.rs:21:16
   |
21 |     let body = entry.response.content.text.ok_or(err_msg("no response body"))?;
   |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot move out of borrowed content

error: aborting due to previous error```

Reply:
Oh right.  Sprinkle some `as_ref()` on it:
```rust
let body = entry.response.content.text.as_ref().ok_or(err_msg("no response body"))?;
```